net-vpn/torguard-bin/torguard-bin-3.96.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user desktop
+
+DESCRIPTION="TorGuard VPN Client"
+HOMEPAGE="https://torguard.net"
+MY_P="torguard-v${PV}-amd64-arch"
+SRC_URI="https://updates.torguard.biz/Software/Linux/${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="sys-apps/iproute2"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+src_unpack() {
+	unpack ${A}
+	unpack "${MY_P}/${MY_P}".tar
+	rm "${MY_P}/${MY_P}".tar || die
+}
+
+src_install() {
+
+	local opt_root="/opt/torguard"
+	dodir ${opt_root}
+
+	insinto $opt_root
+	doins -r "./opt/torguard"/*
+
+	doicon "./usr/share/pixmaps/torguard.png"
+	domenu "./usr/share/applications/torguard.desktop"
+	insinto /usr/share/polkit-1/actions
+	doins "./usr/share/polkit-1/actions/com.torguard.policy"
+	insinto /etc/sudoers.d
+	doins "./etc/sudoers.d/torguard"
```

Comment:
This looks wrong with the local directories respectively dots  from a first view. Please take another look if this is really what you want to achieve. I had no time so far to test your ebuild though and this impression comes just from reading the code.

Reply:
the released package is for archlinux, those're the direct hierarchy of the contents after extracted, maybe a comment should be there to avoid confusing 😂